tests/functional/pv/space_reclaim/test_rbd_reclaimspace_cronjob.py

```diff
@@ -257,7 +265,98 @@ def test_reclaim_space_cronjob_with_annotation(
             pvc_to_chron_job_dict[chron_job_name] == chron_job_schedule
         ), "Reclaim space cron job does not exist, or schedule is not correct"
 
-    def wait_for_cronjobs(self, cronjobs_exist, timeout=60):
+    @tier1
+    @bugzilla("2183444")
+    @polarion_id("OCS-5500")
+    @skipif_hci_provider_and_client
+    @skipif_external_mode
+    def test_modify_scedule_reclaim_spacejob(
+        self,
+        pvc_factory,
+    ):
+        """
+        Test case to check that the schedule to run ReclaimSpaceJob accidentally disabled CronJob is modified
+        if schedule has changed after 100 * new-interval seconds passed since rsCronJob.Status.LastScheduleTime.
+
+        Steps:
+        1. Create a pvc.
+        2. Enable reclaim space operation using Annotating PVC.
+        3. Wait for more than 100 * 1 minutes = ~ 2 hours.
+        4. Modify the schedule to run ReclaimSpaceJob job every minute.
+        5. Grep for error message "delete and recreate reclaimspacecronjob" in csi-addons-controller pod logs.
+        """
+
+        # get random size for pvc
+        ceph_cluster = CephCluster()
+        pvc_size = random.randint(1, int(ceph_cluster.get_ceph_free_capacity()))
+
+        pvc_status = constants.STATUS_BOUND
+        pvc_obj = pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            size=pvc_size,
+            access_mode=constants.ACCESS_MODE_RWO,
+            status=pvc_status,
+            volume_mode=constants.VOLUME_MODE_BLOCK,
+        )
+
+        helpers.wait_for_resource_state(pvc_obj, pvc_status)
+
+        log.info("add reclaimspace.csiaddons.openshift.io/schedule label to PVC ")
+        OCP(kind=constants.PVC, namespace=pvc_obj.project.namespace).annotate(
+            '"reclaimspace.csiaddons.openshift.io/schedule=@midnight"',
+            pvc_obj.name,
+            overwrite=False,
+        )
+
+        log.info("Wait for 2 hrs")
+        time.sleep(6000)
+        pvc_to_chron_job_dict = self.wait_for_cronjobs(
+            True, pvc_obj.project.namespace, 60
+        )
+        assert pvc_to_chron_job_dict, "Reclaim space cron job does not exist"
+
+        chron_job_name = (
+            pvc_obj.get()
+            .get("metadata")
+            .get("annotations")
+            .get("reclaimspace.csiaddons.openshift.io/cronjob")
+        )
+        chron_job_schedule = (
+            pvc_obj.get()
+            .get("metadata")
+            .get("annotations")
+            .get("reclaimspace.csiaddons.openshift.io/schedule")
+        )
+
+        assert (
+            pvc_to_chron_job_dict[chron_job_name] == chron_job_schedule
+        ), "Reclaim space cron job does not exist, or schedule is not correct"
+
+        OCP(kind=constants.PVC, namespace=pvc_obj.project.namespace).annotate(
+            '"reclaimspace.csiaddons.openshift.io/schedule=*/1 * * * *"',
+            pvc_obj.name,
+            overwrite=True,
+        )
+        expected_log = "delete and recreate reclaimspacecronjob"
```

Comment:
IIUC, the fix is to improve the error message. If that is the case, is there any possibility if this can be added to some existing test instead of a new test?